src/main/java/com/google/sps/servlets/AuthFilter.java

```diff
@@ -40,61 +40,42 @@
 import com.google.firebase.auth.FirebaseAuthException;
 import com.google.sps.servlets.MutableHttpServletRequest;
 
```

Comment:
Can you remove the empty line between imports?

Reply:
okay

Reply:
done